account/accounts_geth.go

```diff
@@ -0,0 +1,42 @@
+// +build !nimbus
+
+package account
+
+import (
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/status-im/status-go/account/generator"
+)
+
+// GethManager represents account manager interface.
+type GethManager struct {
+	Manager
+
+	manager *accounts.Manager
+}
+
+// NewManager returns new node account manager.
+func NewManager() *GethManager {
+	m := &GethManager{}
+	m.accountsGenerator = generator.New(m)
```

Comment:
Then here you would need to do:
```suggestion
	m.Manager = &Manager{accountsGenerator: generator.New(m)}
```